src/connection/tests.rs

```diff
@@ -900,7 +904,7 @@ fn from_connect_fetchall_set_get_lazy_panic_executing() {
 
     connection.status = ConnectionStatus::Executing;
     connection.set_lazy(false);
-    assert_eq!(false, connection.lazy);
+    assert!(!connection.lazy);
```

Comment:
I think this assert (and a lots of other ones also) doesn't make sense. We expect that `connection.set_lazy` panics, therefore any assert after that doesn't make sense in my opinion. Not just because it is unreachable when the tests succeeds, but also it does't provide any additional information: when you expect such a serious issue that it causes panic, then it meaningless to expect anything.

Reply:
Nice catch!